lib/code_corps/stripe_service/adapters/stripe_connect_account.ex

```diff
@@ -62,6 +62,36 @@ defmodule CodeCorps.StripeService.Adapters.StripeConnectAccountAdapter do
     {:verification_fields_needed, [:verification, :fields_needed]}
   ]
 
+  @stripe_update_read_only [
```

Comment:
Is this still used for something? I tried searching for it through the file but got nothing.